corehq/apps/reports/static/reports/js/bootstrap5/tabular.js

```diff
@@ -68,6 +68,9 @@ hqDefine("reports/js/bootstrap5/tabular", [
 
     // Handle async reports
     $(document).on('ajaxSuccess', function (e, xhr, ajaxOptions, data) {
+        if (!data || !data.slug) {
```

Comment:
Because `inddex/main` is now importing this file, it gets this event handler, which it doesn't need, and which errors sometimes (presumably because there are ajax requests happening that aren't the same as what this handler expects). But checking for `data.slug` is pretty innocuous and fixes the issue.

Reply:
I think it would be cleaner to put the event handlers in an `init` function, and make code that imports this module explicitly call it, but I wanted to limit changes to core reports as much as possible.

Reply:
What do you think of putting [this comment](https://github.com/dimagi/commcare-hq/pull/35605/commits/493d6b134c97eef869cf6f0ad8c3e39a623c1203#r1909573334) in the code? This seems far away from `inddex/main`, and it might be nice to know why this check was added.

Reply:
Sure - 715c44157c6